ui/storybook/package.json

```diff
@@ -7,7 +7,9 @@
     "storybook:start": "storybook dev --config-dir src/config --port 6006",
     "storybook:clean": "rimraf storybook-static/",
     "storybook:build": "npm run storybook:clean && storybook build --config-dir src/config",
-    "storybook:serve": "npx http-server ./storybook-static",
+    "storybook:serve": "npx http-server --port=6006 ./storybook-static",
+    "storybook:test": "test-storybook --config-dir src/config --stories-json",
```

Comment:
This is primarily used when developing because you can run it against your active storybook.